### ISSUE TITLE
Password update

### ISSUE THREAD

**Issue by tangentfairy-zz** [CoderDojo/community-platform#432]
I can reset my password but if I want to log in and simply change it there is no way to do so

Requested by myself

**Comment by josmas** [comment]
Now you know why! :D

**Comment by clionaderoiste** [comment]
@tangentfairy is this meant to be open?

**Comment by tangentfairy** [comment]
@clionaderoiste yes - ideally we'd just add this to the edit profile section, so that you have to enter: Your current password, and your new password twice to change it.
